src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,99 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final int DEFAULT_COEFFICIENT = 2;
+    private static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    private int threshold;
+    private Node<K, V>[] table;
+    private int size;
+
+    public MyHashMap() {
+        this.table = new Node[DEFAULT_CAPACITY];
+    }
 
     @Override
     public void put(K key, V value) {
-
+        resizer();
+        int bucket = getIndex(key);
+        Node<K,V> node = table[bucket];
+        Node<K,V> newNode = new Node<>(key, value, null);
+        if (node == null) {
+            table[bucket] = newNode;
+        }
+        while (node != null) {
+            if (Objects.equals(key, node.key)) {
+                node.value = value;
+                return;
+            } else if (node.next == null) {
+                node.next = newNode;
+                break;
+            }
+            node = node.next;
+        }
+        size++;
```

Comment:
```suggestion
        for ( ;node != null; node = node.next;) {
            if (Objects.equals(key, node.key)) {
                node.value = value;
                return;
            } 
            if (node.next == null) {
                node.next = newNode;
                break;
            }
        }
        size++;
```
else after return is redundant